packages/bot-web-ui/src/components/dashboard/quick-strategy/quick-strategy-components/data/data-fields.ts

```diff
@@ -0,0 +1,97 @@
+import { data_uniq_input_obj, common_inputs_properties } from '..';
+import { TSelectsFieldNames, TDropdownItems, TInputBaseFields, TInputsFieldNames } from '../../quick-strategy.types';
+import { TDataUniqInput } from './data-uniq-input-obj';
+import { TCommonInputsProperties } from './common-input-properties';
+
+const getMessage = (property: string) => `The bot will stop trading if your total ${property} exceeds this amount.`;
+
+export type TDataFields = {
+    id: string;
+    name: string;
+    field_name: TSelectsFieldNames | TInputsFieldNames;
+    className?: string;
+    input_value: TDropdownItems | TInputBaseFields;
+    label: string;
+    placeholder?: string;
+    trailing_icon_message?: string;
+    is_able_disabled?: boolean;
+} & Readonly<TCommonInputsProperties>;
+
+const data_fields: ReadonlyArray<TDataFields | ReadonlyArray<TDataUniqInput>> = [
+    {
+        id: 'base__types-strategies',
```

Comment:
id and name both are the same? what's the use of having both ? and is it possible to change the format?

Reply:
@sara-fs yes, we can remove the id or name, right.  if we remove 'base__' for example, then we need to add any other property to know what is the type of field...

Reply:
done